configure.ac

```diff
@@ -3514,6 +3514,7 @@ else
 fi
 AC_SUBST(PO2HTML)
 AC_SUBST(PO2TEXT)
+AM_CONDITIONAL(ENABLE_TRANSLATION, test "x${enable_translation:=no}" = "xyes")
 
```

Comment:
> PR description:  The following warning is still shown, even in explicit --disable-translation builds:

    WARNING: Translation is disabled.

I believe that "make install" should not _warn_ admins that explicitly disabled translations that they are getting exactly what they explicitly asked for during `./configure`. I consider that problem out of this PR scope, but if others insist on it being addressed in this PR, I will do that instead. If you think that warning should stay as it is, please say so -- such a disagreement would confirm that the problem should be kept outside this PR scope.

Reply:
Agreed that it needs to change, but not insisting on it for this.